rhpam-config/7.12.1/common.yaml

```diff
@@ -1105,22 +1105,22 @@ servers:
                   #[[end]]
                   livenessProbe:
                     httpGet:
-                      path: /services/rest/server/healthcheck
-                      port: 8080
-                      scheme: HTTP
-                    initialDelaySeconds: 180
-                    timeoutSeconds: 2
-                    periodSeconds: 15
-                    failureThreshold: 3
+                      path: "[[.LivenessProbe.Handler.HTTPGet.Path]]"
```

Comment:
I liked your approach, isn't better remove completely remove it from templates and handle it with default values and based on values provided on the CR?

Reply:
Since is a specific request just from one source, I'd prefer to leave as usual as possible